### ISSUE TITLE
The doubt about random results

### ISSUE THREAD

**Issue by jimmy-dq** [silverbottlep/meta_trackers#5]
Hi, @silverbottlep , thaks for your excellent work. I just run your code (meta_sdnet/run_tracker.py) and find that you fix the seed in the experiment, like  np.random.seed(1), torch.manual_seed(2) and torch.cuda.manual_seed(3). But i still find that the result is random, could you explain this ?

**Comment by zhanglichao** [comment]
Yes, I also met the same problem. Is there any solutions for that?

best
